### PR TITLE
feat(financial-statements-inao): financial statements inao template init

### PR DESCRIPTION
**Pull request by HelgiHH**
## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — APPROVED** [review]
The 4 core owned files LGTM

**Review by Thorrik58 — CHANGES_REQUESTED** [review]

File: libs/application/templates/financial-statements-inao/src/shared/constants.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum ApiActions {
+  createApplication = 'createApplication',
+  doStuffThatFails = 'doStuffThatFails',
+  completeApplication = 'completeApplication',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/shared/constants.ts:4]: Is this used anywhere? Should probably be removed if not.

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          ],
+        }),
+        buildDescriptionField({
+          id: 'final',
+          title: m.thankYou,
+          description: (application) => {
+            const sendApplicationActionResult =
+              application.externalData[ApiActions.createApplication]
+
+            let id = 'unknown'
```
[the remark this answers is not in the document]
Comment (HelgiHH) [reply libs/application/templates/financial-statements-inao/src/forms/application.ts:160]: this is from the example project, forgot to remove. my bad

File: libs/application/templates/financial-statements-inao/src/shared/constants.ts (hunk 1 above)
Comment (HelgiHH) [reply libs/application/templates/financial-statements-inao/src/shared/constants.ts]: removed

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        buildSubSection({
+          id: 'income',
+          title: m.income,
+          children: [
+            buildMultiField({
+              title: '',
+              children: [
+                buildTextField({
+                  id: 'income.donations',
+                  title: m.donations,
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:100]: Should this be defined with varian: 'number'?

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+                  width: 'half',
+                }),
+                buildTextField({
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:103]: same as above

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+                  id: 'income.personal',
+                  title: m.personalIncome,
+                  width: 'half',
+                }),
+                buildTextField({
+                  id: 'income.capital',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:109]: same as above

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+      id: 'info',
+      title: m.infoSection,
+      children: [
+        buildMultiField({
+          id: 'about',
+          title: m.about,
+          children: [
+            buildTextField({
+              id: 'about.email',
+              title: m.email,
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:54]: Shouldn't this be defined with variant: 'email' ?

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+              width: 'half',
+            }),
+            buildTextField({
+              id: 'about.phoneNumber',
+              title: m.phoneNumber,
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:59]: Shouldn't this be defined with variant: 'tel' ?

File: libs/application/templates/financial-statements-inao/src/forms/application.ts [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+            buildMultiField({
+              title: '',
+              children: [
+                buildTextField({
+                  id: 'properties.short',
+                  title: m.propertiesShort,
+                  width: 'half',
+                }),
+                buildTextField({
+                  id: 'properties.cash',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/application.ts:83]: Shouldn't this be defined with variant: number/currency ?

File: libs/application/templates/financial-statements-inao/src/forms/done.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+export const done: Form = buildForm({
+  id: 'done',
+  title: 'Umsókn móttekin',
+  mode: FormModes.APPLYING,
+  children: [
+    buildCustomField({
+      id: 'overview',
+      component: 'Success',
+      title: 'Umsókn móttekin',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/forms/done.ts:16]: Should probably have a translation string.

File: libs/application/templates/financial-statements-inao/src/lib/messages.ts [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+      'Vinsamlegast lestu yfir umsóknina og vertu viss um að allar upplýsingar séu rétt uppgefnar. Að loknum yfirlestri getur þú sent inn umsóknina.',
+    description: 'Description for overview section',
+  },
+  success: {
+    id: 'fsn.application:success',
+    defaultMessage: 'Skil á árskýrslu mótteking',
+    description: 'financial statement success',
+  },
+  dataCollectionCheckboxLabel: {
+    id: 'ps.application:dataCollectionCheckboxLabel',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/lib/messages.ts:151]: wrong id, should probably be fsn.application not ps

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Some issues with ux and translations.

File: libs/application/templates/financial-statements-inao/src/lib/messages.ts [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+    defaultMessage: 'Senda umsókn',
+    description: 'Send application',
+  },
+  overviewTitle: {
+    id: 'fsn.application:overviewTitle',
+    defaultMessage: 'Yfirlit',
+    description: 'Title for overview section',
+  },
+  overviewSectionTitle: {
+    id: 'fsn.application:overviewTitle',
```
Comment (Thorrik58) [inline libs/application/templates/financial-statements-inao/src/lib/messages.ts:135]: This isnt an unique id and is going to clash with the overviewTitle once you run extract strings on your application. Which you cant now as you dont have the application defined in the workspace.json

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Minor comments. Looking good tho.

**Review by Thorrik58 — APPROVED** [review]
LGTM

**Pull request merged**